### DISCUSSION TITLE
Shelf V1 Vision + MVP (KyllVP)

### DISCUSSION THREAD

**Discussion by jurrejansen** [Shelf-nu/shelf.nu#30] [General]
The list below shows our V1 ambition. Below you can find the KyllVP (MVP) features.

## Authorization

- Log In
- Sign Up
- Forgot Password
- Magic Link

## Settings

- My details
- - First name
- - Last name
- - Email
- - Username
- - Profile picture
- ~~Items~~
- ~~Team~~
- ~~Account Plan~~
- ~~Billing~~

## Asset Creation and Management:

- Create Assets
- ~~Archive Assets~~
- Asset index
- Assign Asset Category
- ~~Assign Asset Tags~~
- ~~Assign Asset Owner~~
- Assign image to asset
- ~~Assign Depreciation to asset~~
- Add asset descriptions
- Filter by Category
- ~~Filter by Tags~~
- ~~Filter by Owner~~
- ~~Filter by Location~~
- ~~Multi-select Assets~~
- Edit Asset
- Edit Asset image

## Asset Sorting and Search

- Full Asset Database Search
- - Search by name
- ~~Sort by Alphabetic~~
- ~~Sort by Newest~~
- ~~Sort by Oldest~~
- ~~Sort by Category~~
- ~~Sort by Owner~~
- ~~Sort by A-Z~~
- ~~Sort by Z-A~~

## Asset Activity

- ~~See Asset Tag Scan Activity~~
- ~~See full timeline of Asset Tag Scans~~
- See last item scan location on a map showing X and Y coordinates is good.

## Asset Tag Printing and Export:

- ~~Print Asset Tags~~
- ~~Batch Print QR Tags~~
- Print Small QR Asset Tags
- Print Medium QR Asset Tags
- Print Large QR Asset Tags
- Print Extra Large QR Asset Tags
- ~~Full asset database export~~
- ~~Full asset detail export~~

## Asset Management Apps

- ~~ Mobile App ~~
- ~~ Desktop App ~~
- Responsive web application

## Asset Commenting

- Add notes to items

**Comment by carlosvirreira** [comment]
## The MVP (KyllVP)

### Authorization
  - [x] :closed_lock_with_key: Log In
  - [x] :clipboard: Sign Up
  - [x] :key: Forgot Password
  - [x] :link: Magic Link

### Settings
  - [x] :bust_in_silhouette: My details
    - [x] :label: First name
    - [x] :label: Last name
    - [x] :email: Email
    - [x] :label: Username
    - [x] :framed_picture: Profile picture

### Asset Creation and Management:
  - [x] :heavy_plus_sign: Create Assets
  - [x] :file_folder: Asset index
  - [x] :label: Assign Asset Category
  - [x] :framed_picture: Assign image to asset
  - [x] :pencil: Add asset descriptions
  - [x] :mag_right: Filter by Category
  - [x] :pencil2: Edit Asset
  - [x] :framed_picture: Edit Asset image
  - [x] 👨‍💻 #100 

### Asset Sorting and Search
  - [x] :mag: Full Asset Database Search
    - [x] :abc: Search by name

### Asset Activity
  - [x] :world_map: See last item scan location on a map showing X and Y coordinates is good.

### Asset Tag Printing and Export:
  - [x] :printer: Print Small QR Asset Tags
  - [x] :printer: Print Medium QR Asset Tags
  - [x] :printer: Print Large QR Asset Tags
  - [x] :printer: Print Extra Large QR Asset Tags

### Asset Management Apps:
  - [x] :iphone: Responsive web application

### Asset Commenting:
  - [x] :notebook: Add notes to items

**Reply by DonKoko** [reply]
THanks

**Comment by carlosvirreira** [comment]
Request from potential customer talk TM is Import via CSV (they have an asset list and for them it saves a lot of time to do like this)
